### ISSUE TITLE
strange warnings on startup on Linux Korora (Fedora-based)

### ISSUE THREAD

**Issue by jlp78** [synergy/synergy#4818]
I have observed the following messages when starting synergy as a client on Linux talking to a Mac OS X server.  NOTE: synergy seems to run just fine... these are definitely only warnings.

QGtkStyle could not resolve GTK. Make sure you have installed the proper libraries.
*** WARNING *** The program 'synergy' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/avahi-compat?s=libdns_sd&e=synergy>
*** WARNING *** The program 'synergy' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/avahi-compat?s=libdns_sd&e=synergy&f=DNSServiceRegister>

**Comment by gsalvati** [comment]
Same for me. Any news?

**Comment by speaker** [comment]
Duplicate #4672

**Comment by jlp78** [comment]
The linked issue covers the Avahi messages, but not this one:

QGtkStyle could not resolve GTK. Make sure you have installed the proper libraries.

I figured out what happened here... I had installed the i686 version of synergy on my x86_64 architecture machine.  Most of the dependencies were properly installed, but the i686 version of libgnomeui was not.  Installing it (yum install libgnomeui.i686) made the above error go away, but it was replaced with a new message complaining about the adwaita theme.  Installing the i686 version of the adwaita-gtk2-theme made the new warning messages go away as well.

You might want to check the rpm spec file and add libgnomeui and adwaita-gtk2-theme as dependencies to avoid these messages.  This is LOW priority, though, as it seems to run just fine even without these packages installed.
